Remotion/Data/DomainObjects.UnitTests/Factories/BaseConfiguration.cs

```diff
@@ -118,5 +124,28 @@ public FakeDomainObjectsConfiguration GetDomainObjectsConfiguration ()
     {
       return new FakeDomainObjectsConfiguration(_mappingLoaderConfiguration, _storageConfiguration, _queryConfiguration);
     }
+
+    public void DisableConnectionsAccess ()
```

Comment:
It doesn't need to be `DisableDatabaseAccess` from my own suggestion, however `DisableConnectionsAccess` doesn't work for me because it doesn't disable the access to the "connections", what a "connection" is. It disables the access to the databases identified by the connection strings. I would like to a have more targeted method name.